ios/Classes/RSIShareViewController.swift

```diff
@@ -67,6 +67,15 @@ open class RSIShareViewController: SLComposeServiceViewController {
                                                          index: index,
                                                          content: content)
                                     }
+                                case .pkpass:
+                                    if let raw = data as? NSData{
```

Comment:
personally I would prefer to have it in brackets
```suggestion
                                    if (let raw = data as? NSData) {
```
or at least a space before the curly bracket?

Reply:
Okay the space i can inderstand, i didn't find the autoformat in XCode, but if we added brackets here, we would have to add in in every case, which would make it imcompatible, if we ever wanted to upmerge this